appveyor.yml

```diff
@@ -24,7 +24,10 @@ build_script:
   - conda update conda
   - conda install conda-build
   - ps: if(-not $env:APPVEYOR_PULL_REQUEST_NUMBER) { conda install anaconda-client }
-  - conda build conda.recipe --channel dynd/channel/dev
+  - conda build conda\ndt.recipe --channel dynd/channel/dev
+  - rd /q /s build dist dynd.ndt.egg-info
+  - echo dynd.ndt >> requirements.txt
```

Comment:
Sorry about the batch misinformation there. I forgot that even though this doesn't always work it still works fine in most cases. http://stackoverflow.com/questions/7225630/how-to-echo-2-no-quotes-to-a-file-from-a-batch-script was the issue I was thinking of, and it doesn't apply in this case.

Thanks for handling this in setup.py.